controllers/useraccount/useraccount_controller_test.go

```diff
@@ -626,9 +623,9 @@ func TestReconcile(t *testing.T) {
 			Get()
 
 		// Set the deletionTimestamp
-		now := metav1.NewTime(time.Now())
-		userAcc.DeletionTimestamp = &now
-		err = r.Client.Update(context.TODO(), userAcc)
+		//now := metav1.NewTime(time.Now())
+		//userAcc.DeletionTimestamp = &now
```

Comment:
```suggestion
```

Reply:
done in [0b5289e](https://github.com/codeready-toolchain/member-operator/pull/551/commits/0b5289e17f8337c2eeba756dd94a524b87163fd0), thanks